workflow/snakemake_rules/CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Developer guide
+
+## Snakemake formatting
+
+We use [`snakefmt`](https://github.com/snakemake/snakefmt) to ensure consistency in style across Snakemake files in this project.
+
+### Installing
+
+- Using mamba/bioconda:
+
+```bash
+mamba install -c bioconda snakefmt
+```
+
+- Using pip:
+
+```bash
+pip install snakefmt
+```
```

Comment:
_(non-blocking)_

My preference is to link out to https://github.com/snakemake/snakefmt#install, but I can see how this is easier for the reader to follow along.